docs/API.md

```
@@ -0,0 +1,541 @@
+# jsont API Documentation
+
+jsont の主要な API、型定義、および拡張可能なインターフェースについて説明します。
+
+## 📋 目次
+
+- [基本型定義](#基本型定義)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)